app/utils/servers.js

```diff
@@ -0,0 +1,3 @@
+export function getServerPlayers(server) {
+    return server.variables.players ? server.variables.players : server.players;
```

Comment:
If you're introducing a utility method for this I'd suggest also changing the server component to use this method to calculate how many players to show, and incorporating the entire logic here (including the spectator adjustments). I'd also keep the NaN check.